src/xargs/mod.rs

```diff
@@ -40,6 +41,7 @@ struct Options {
     max_lines: Option<usize>,
     no_run_if_empty: bool,
     null: bool,
+    replace: Option<String>,
```

Comment:
Maybe we should be using a `&[u8]` here? GNU probably accepts invalid utf-8. Otherwise, this is looking good.

Reply:
i just tried and it requires adding lifetime everywhere
is it worst it?